public/static/docs/tutorials/deep/sharing-data.md

```diff
@@ -51,13 +51,14 @@ the master branch:
 
 ```dvc
 $ git clone https://github.com/dmpetrov/new_tag_classifier.git
+$ cd new_tag_classifier
 $ dvc pull
 ```
```

Comment:
Here's the fix to https://github.com/iterative/dvc.org/pull/938#issuecomment-576884513 (Dmitry restored the repo) `cd` was also missing.

Reply:
switched to example-get-started like was suggested?

Reply:
Yes. Will be addressed in #946.

Reply:
👍

Reply:
OK it was a small change so I did it here as well (54bcf5e5).